### PR TITLE
feat(accessibility): high contrast accessibility

### PR DESCRIPTION
**Pull request by dalipkumar703**
## Description
This PR add accessibility for list item and radio/checkbox input.

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: core/scss/components/button/button.scss [hunk 1]
```diff
@@ -26,7 +26,7 @@
       $button__transition-speed,
       ease
     );
-
```
Comment (bfbiggs) [inline core/scss/components/button/button.scss:29]: Unecessary changes here.  Please remove these.

File: core/scss/components/list-item/list-item.scss [hunk 2]
```diff
@@ -8,6 +8,7 @@
     @include list-item--dark;
     @include list-item--contrast;
 
+    border: 1px solid transparent;
```
Comment (bfbiggs) [inline core/scss/components/list-item/list-item.scss:11]: Seems this should be handled in the core list-item mixin.

**Comment by bfbiggs** [comment]
Commit doesn't match change.

File: core/scss/components/button/button.scss (hunk 1 above)
Comment (dalipkumar703) [reply core/scss/components/button/button.scss]: > 
> 
> Commit mesage doesn't match change.

I updated PR with above request change. You can have a look.

File: core/scss/components/list-item/mixins.scss [hunk 3]
```diff
@@ -11,6 +11,7 @@
   margin-left: $margin-left;
   order: $order;
   flex: $flex-grow $flex-shrink $flex-basis;
+  border: 1px solid transparent;
```
Comment (bfbiggs) [inline core/scss/components/list-item/mixins.scss:14]: Not sure we would want this setting in the list-item mixin:

1. No variables to change it
2. Already have a list-item-color which might more appropriately handle border color.

Comment (dalipkumar703) [reply core/scss/components/list-item/mixins.scss:14]: I guess list-item--color not setting border 1px which might not show border in window high contrast mode.

**Pull request closed without merging**